packages/jest-config/jest.config.ts

```diff
@@ -0,0 +1,38 @@
+import type { Config } from '@jest/types';
+import deepmerge from 'deepmerge';
+
+const baseConfig: Config.InitialOptions = Object.freeze({
+  preset: 'ts-jest',
+  transform: {
+    // Enable JS/JSX transformation
+    '\\.(ts|js)x?$': 'ts-jest',
+    '\\.(svg)$': 'jest-transform-stub',
```

Comment:
ditto: same for svg is UI exclusive.

Reply:
Ditto. And since it's a dev config, I'm thinking about the tolerance of extra dependencies, just like eslint plugins.